### ISSUE TITLE
Game screen isn't resizing with the window

### ISSUE THREAD

**Issue by Simonx22** [PSI-Rockin/CorgiDS#7]
Like the title is saying, it doesn't resize. This also happens with the configuration window. Tested on Ubuntu but I can also reproduce this on Windows.
Screenshots:
![image](https://user-images.githubusercontent.com/26326692/34322741-7f762042-e830-11e7-85cc-8443befef588.png)
![image](https://user-images.githubusercontent.com/26326692/34322743-861b52f0-e830-11e7-8c5a-b6b03736e0c0.png)
![image](https://user-images.githubusercontent.com/26326692/34322748-a61d8438-e830-11e7-9c77-23af9d72968c.png)

**Comment by PSI-Rockin** [comment]
Yeah, that's normal. CorgiDS directly displays the framebuffer without resizing it.
Fixing it probably wouldn't be difficult, but it's low priority until I get the Windows stuff working.
